string addr_low
string node_id
uint16 query_id
time last_reported
uint8 rssi